requirements_extra.txt

```diff
@@ -2,4 +2,5 @@ netCDF4
 ipywidgets
 ipython
 geojson
+pymetalink
```

Comment:
pymetalink only available on GitHub? I haven't seen a package on pypi or conda.

Reply:
That's my understanding. It does not seem to be very active, but it works.

Reply:
You could add the pip+github install in the conda `environment.yml`.

Reply:
Done.